src/main/java/com/training/utilities/JsonReaderUtility.java

```diff
@@ -0,0 +1,32 @@
+package com.training.utilities;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.util.Map;
+
+public class JsonReaderUtility {
+
+    public Map getJsonHashMap() {
```

Comment:
Make this return a generic Map with both value and key as strings. Then you can avoid the `toString()` in your tests.